.travis.yml

```diff
@@ -6,5 +6,10 @@ install:
     - source utils/init-environment
     - pip install codecov
 script: pyproctor
+services:
+    - elasticsearch
+# This is required for elasticsearch
+before_script:
+    - sleep 10
```

Comment:
Unfortunately in travic-ci docs it says to add this. https://docs.travis-ci.com/user/database-setup/#ElasticSearch